frontend/HelperFuncs/utils.js

```diff
@@ -67,4 +67,47 @@ export const logIn = async (userData) => {
 }
 
 
+/*
+helper function to fetch list of jobs matching [searchTerm] from the 
+back4app BLS occupations list endpoint
+*/
+export const fetchJobTitles = async (searchTerm) => {
+    const where = encodeURIComponent(JSON.stringify({
+        "title": {
+            "$regex": searchTerm,
+            "$options": "i"
+        }
+    }));
+    const headers = {
+        'X-Parse-Application-Id': import.meta.env.VITE_API_BLS_ID,
+        'X-Parse-REST-API-Key': import.meta.env.VITE_API_BLS_KEY
+    };
+    return fetch(`https://parseapi.back4app.com/classes/Occupations_Job?limit=10&keys=title&where=${where}`, {
+        headers: headers
+    })
+        .then(response => response.json())
+        .then(data => {
+            if (data.results) {
+                // map the fetched options into the value-label form react-select uses
+                const jobOptions = data.results.map(job => ({ value: job.title, label: job.title }));
+                return(jobOptions)
+            } else {
+                console.log('No results found');
```

Comment:
Don't we need to return something in the else clause too? Or throw to trigger the catch

Reply:
Good catch! I must've just forgot to add something more permanent there while debugging. This will be fixed in PR #15